Formula/omero53.rb

```diff
@@ -0,0 +1,86 @@
+require 'formula'
+
+class Omero53 < Formula
+  homepage 'http://www.openmicroscopy.org/site/products/omero'
+
+  url 'http://downloads.openmicroscopy.org/omero/5.3.0-m8/artifacts/openmicroscopy-5.3.0-m8.zip'
+  sha256 '9cffaadceb9245748686774c971da7020f10dcb798ec63b28cb494f022f87fe6'
+
+  option 'with-cpp', 'Build OmeroCpp libraries.'
+  option 'with-ice36', 'Use Ice 3.6.'
```

Comment:
Is that supposed to be `with-ice35` as per the dependency declaration below? At the moment, this current version of the formula might unconditionally build with Ice 3.6.